src/client.js

```diff
@@ -54,6 +54,13 @@ class SkynetClient {
     // SkyDB
     this.db = {
       setDataLink: browserClient.db.setDataLink.bind(browserClient),
+      setJSON: nodejs_db_setJSON.bind(this),
+      getJSON: browserClient.db.getJSON.bind(browserClient),
+      deleteJSON: browserClient.db.deleteJSON.bind(browserClient),
+//      getEntryData: browserClient.db.getEntryData.bind(browserClient),
+//      setEntryData: nodejs_db_setEntryData.bind(this),
+//      deleteEntryData: browserClient.db.deleteEntryData.bind(browserClient),
+//      getRawBytes: browserClient.db.getRawBytes.bind(browserClient),
```

Comment:
You can add these if you have a need for them, but I would make sure that they are tested, first.

Reply:
I tested this and looked at and understand in the "Skynet js" code! I think there should be no problems!
Except the "Trawbytes" function.
I had integrated the 4 functions for the test for you.
Since I have already tapped the last few days, I have now reached DBV2, and I also try my luck with the "DBV2.Setjson" function, but is a little different!
Remaining functions of "DBV2" already tested and ok!
But we have to get "DBV2.Setjson" first!